Backend/app/user/user_service.py

```diff
@@ -201,6 +201,87 @@ def upgrade_to_regular_licence(
         except sqlite3.IntegrityError:
             return {"error": "No se pudo actualizar el usuario"}
 
+    # -------------------------------------------
```

Comment:
Maybe you could move all of these new functions to a different file, user_service is already quite bloated.